storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst

```diff
@@ -103,21 +113,16 @@ The procedure is analogous to :ref:`restoring individual tables
    $ xtrabackup --prepare --export --target-dir=/path/to/partial/backup
 
 
-When you use the :option:`xtrabackup --prepare` option on a partial backup, you
-will see warnings about tables that don't exist. This is because these tables
-exist in the data dictionary inside InnoDB, but the corresponding :term:`.ibd`
-files don't exist. They were not copied into the backup directory. These tables
-will be removed from the data dictionary, and when you restore the backup and
-start InnoDB, they will no longer exist and will not cause any errors or
-warnings to be printed to the log file.
+If you use the :option:`--prepare` option on a partial backup, and you see warnings about tables that do not exist, these tables exist in the data dictionary inside InnoDB, but their corresponding :term:`.ibd` files do not exist. These files were not copied into the backup directory. The tables are removed from the data dictionary during the process and do not cause errors or warning in subsequent backups and restores.
 
-An example of the error message you will see during the prepare phase follows. ::
+An error message example during the prepare phase follows. ::
 
   InnoDB: Reading tablespace information from the .ibd files...
   101107 22:31:30  InnoDB: Error: table 'test1/t'
   InnoDB: in InnoDB data dictionary has tablespace id 6,
   InnoDB: but tablespace with that id or name does not exist. It will be removed from data dictionary.
 
+
```

Comment:
For the lines below:

Restoring Partial Backups
================================================================================

Restoring should be done by :ref:`restoring individual tables
<restoring_individual_tables>` in the partial backup to the server.
It can also be done by copying back the prepared backup to a "clean"
:term:`datadir` (in that case, make sure to include the ``mysql``
database) to the datadir your are moving the backup to. System database can be created with:
.. code-block:: bash
   $ sudo mysqld --initialize --user=mysql

Once you start the server, you might see mysql complaining about missing tablespaces:

```
2021-07-19T12:42:11.077200Z 1 [Warning] [MY-012351] [InnoDB] Tablespace 4, name 'test1/t1', file './d2/test1.ibd' is missing!
2021-07-19T12:42:11.077300Z 1 [Warning] [MY-012351] [InnoDB] Tablespace 4, name 'test1/t1', file './d2/test1.ibd' is missing!
```

In order to clean the orphan database from the data dictionary, you will have to manually create the missing database directory then DROP it from the server:

```
$ mkdir /var/lib/mysql/test1/d2
mysql> DROP DATABASE d2;
Query OK, 2 rows affected (0.05 sec) 
```

Reply:
Added.